README.md

```diff
@@ -14,7 +14,7 @@ ebx-github-cycletime itself is open source software released under the terms of
 
 ## Getting Started
 
-At the time of writing the application is intended to be run from source (>=Java 14). Use the 
```

Comment:
Would be nice to have the JIRA steps in there as well, Those in the email were perfect